nx.json

```diff
@@ -141,14 +141,19 @@
       "options": {
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33028&commitId=2e9d98341ccdb6042cad013cd7181cd148f97430&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.chromium.png|3| Changed|
|Avatar Converged.Badge Mask RTL.chromium.png|10| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>